<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="new_scene"/>
    <addaction name="open_scene"/>
    <addaction name="separator"/>
    <addaction name="save_scene"/>
    <addaction name="save_as_scene"/>
    <addaction name="separator"/>
    <addaction name="close_scene"/>
    <addaction name="close_all_scene"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menuCwtyf">
    <property name="title">
     <string>Сцена</string>
    </property>
    <addaction name="set_projection_scene"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Создать</string>
    </property>
    <addaction name="create_GameObject2D"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuCwtyf"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>Главная панель инструментов</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="new_scene"/>
   <addaction name="open_scene"/>
   <addaction name="save_scene"/>
  </widget>
  <widget class="QDockWidget" name="panel_shader">
   <property name="windowTitle">
    <string>Менеджер шейдеров</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTreeWidget" name="treeWidget_shader">
       <column>
        <property name="text">
         <string>ID key</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Путь вершинного шейдера</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Путь фрагментного шейдера</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_add_shader">
       <property name="text">
        <string>Добавить шейдер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_del_shader">
       <property name="text">
        <string>Удалить шейдер</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="panel_mesh">
   <property name="windowTitle">
    <string>Менеджер мешей</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTreeWidget" name="treeWidget_mesh">
       <column>
        <property name="text">
         <string>ID key</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Путь</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_add_mesh">
       <property name="text">
        <string>Добавить мешь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_del_mesh">
       <property name="text">
        <string>Удалить мешь</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="panel_texture">
   <property name="windowTitle">
    <string>Менеджер текстур</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTreeWidget" name="treeWidget_texture">
       <column>
        <property name="text">
         <string>ID key</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Путь</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_add_texture">
       <property name="text">
        <string>Добавить текстуру</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_del_texture">
       <property name="text">
        <string>Удалить текстуру</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="panel_sprite">
   <property name="windowTitle">
    <string>Менеджер спрайтов</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QTreeWidget" name="treeWidget_sprite">
       <column>
        <property name="text">
         <string>ID key</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID key shader</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID key mesh</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID key texture</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_add_sprite">
       <property name="text">
        <string>Добавить спрайт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_del_sprite">
       <property name="text">
        <string>Удалить спрайт</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="panel_gameobject">
   <property name="windowTitle">
    <string>Игровые объекты</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QTreeWidget" name="treeWidget_gameobject">
       <column>
        <property name="text">
         <string>Name key</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Удалить объект</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="panel_property">
   <property name="windowTitle">
    <string>Параметры</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_10">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeWidget_property">
       <column>
        <property name="text">
         <string>Параметр</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Значение</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_add_property">
       <property name="text">
        <string>Добавить параметр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_del_property">
       <property name="text">
        <string>Удалить параметр</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="new_scene">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/new_scene.png</normaloff>:/Resources/new_scene.png</iconset>
   </property>
   <property name="text">
    <string>Создать...</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="open_scene">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/open_scene.png</normaloff>:/Resources/open_scene.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="save_scene">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/save_scene.png</normaloff>:/Resources/save_scene.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="save_as_scene">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="close_scene">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="close_all_scene">
   <property name="text">
    <string>Закрыть всё</string>
   </property>
  </action>
  <action name="set_projection_scene">
   <property name="text">
    <string>Настройка проекции</string>
   </property>
  </action>
  <action name="create_GameObject2D">
   <property name="text">
    <string>GameObject2D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
